sql/report_queries/costs/orders.sql

```diff
@@ -0,0 +1,203 @@
+subscription: can be set for "subscription to," subscription from" dates; subscription data only shows for One-Time order type
+workflow status: can be set to pending, open, or closed
+tags: local field with custom settings; can set up to 3 tag filters	
+
+OPTIONAL FIELDS
+Acquistion unit id and acquisition unit name: these fields will only show data if the institution has elected to use them.
+Subscription data entry is optional for institutions.
+
+STILL IN PROGRESS
+-need to add paymentDate field from invoices data array when this field is available
+
+*/
+ 
+--updated to use date
+WITH parameters AS (
+    SELECT
+        '2000-01-01'::DATE AS start_date,
+        '2021-01-01'::DATE AS end_date,
+        ''::VARCHAR AS workflow_status,
+        ''::VARCHAR AS order_type,
+        -- Please comment/uncomment one pair the these NULL parameters if you want to define the range of active subscriptions
+        NULL::DATE AS subscription_from_date,
+        NULL::DATE AS subscription_to_date,
+        --'2000-01-01' :: DATE AS subscription_from_date,
+        --'2021-01-01' :: DATE AS subscription_to_date,
+        ''::VARCHAR AS tags_filter1, -- select 'your first local tag' or leave blank for all. You can use %% as wildcards.
+        ''::VARCHAR AS tags_filter2, -- select 'your second local tag' or leave blank for all. You can use %% as wildcards.
+        ''::VARCHAR AS tags_filter3 --select 'your third local tag' or leave blank for all. You can use %% as wildcards.
+),
+--subquery for po_lines_detail
+po_lines_detail AS (
+    SELECT
+        pol.id AS "po_line_id",
+        pol.po_line_number AS "po_line_number",
+        poltags.pol_tag AS "purchase_order_tag",
+        pol.description AS "purchase_order_description",
+        pol.acquisition_method AS "purchase_order_acquisition_method",
+        polsubdtl.pol_subscription_from AS "pol_subscription_from",
+        polsubdtl.pol_subscription_to AS "pol_subscription_to",
+        polsubdtl.pol_subscription_interval AS "pol_subscription_interval",
+        polermat.pol_er_mat_type_name AS "purchase_order_elec_material_type_name",
+        polphysmat.pol_mat_type_name AS "purchase_order_phys_material_type_name",
+        --pol.instance_id AS "instance_id",
+        pol.purchase_order_id AS "purchase_order_id"
+    FROM
+        po_lines AS pol
+        LEFT JOIN folio_reporting.po_lines_er_mat_type AS polermat ON pol.id = polermat.pol_id
+        LEFT JOIN folio_reporting.po_lines_phys_mat_type AS polphysmat ON pol.id = polphysmat.pol_id
+        LEFT JOIN folio_reporting.po_lines_tags AS poltags ON pol.id = poltags.pol_id
+        LEFT JOIN folio_reporting.po_lines_details_subscription AS polsubdtl ON pol.id = polsubdtl.pol_id
+),
+--subquery for po_purchase_order_detail
+po_purchase_order_detail AS (
+    SELECT
+        podtl.id AS "po_detail_id",
+        podtl.order_type AS "po_order_type",
+        podtl.po_number AS "po_number",
+        podtl.date_ordered AS "po_date_ordered",
```

Comment:
`json_extract_path_text(podtl.data, 'poDateOrdered')`

Reply:
This is fixed.